EventsExpress.Core/Services/EventScheduleService.cs

```diff
@@ -47,15 +48,15 @@ public async Task<OperationResult> Create(EventScheduleDTO eventScheduleDTO)
                 var result = Insert(eventScheduleEntity);
                 await _context.SaveChangesAsync();
 
-                return new OperationResult(true, "Create new EventSchedule", result.Id.ToString());
+                return result.Id;
             }
             catch (Exception ex)
```

Comment:
Do not catch generic exceptions and convert them to our exception. Remove **ALL** such occurrences please.